paymentsheet/src/main/java/com/stripe/android/customersheet/InternalCustomerSheetResult.kt

```diff
@@ -43,7 +43,7 @@ internal sealed class InternalCustomerSheetResult : Parcelable {
             paymentOptionFactory: PaymentOptionFactory,
         ): CustomerSheetResult {
             return CustomerSheetResult.Canceled(
-                selection = paymentSelection?.toPaymentOptionSelection(paymentOptionFactory)
+                selection = paymentSelection?.toPaymentOptionSelection(paymentOptionFactory, canUseGooglePay = true)
```

Comment:
I make this assumption since we calculate this inside of `CustomerSheet`. Maybe I `canUseGooglePay` default to true in this function or make separate functions for both `CustomerSheet` and `InternalCustomerSheetResult`.